pyproject.toml

```diff
@@ -20,15 +20,26 @@ keywords = ["invenio", "edugain"]
 license = {file = "LICENSE"}
 readme = "README.md"
 requires-python = ">=3.9"
-dependencies = []
+dependencies = [
+  "invenio_db>=2.0.0,<3.0.0",
```

Comment:
why is invenio-db here and in the test dependency list? should one place not be enough?

Reply:
in the test dependency-list it's `invenio-db[postgresql]`  (*with* the extra `[postgresql]`)
in the common dependency-list it's `invenio-db` (*without* the extra `[postgresql]`)
so common dependency is db-agnostic, it doesn't install specific db-packages you may not need

this mirrors invenio's dependency-configuration
e.g. [`invenio-jobs`](https://github.com/inveniosoftware/invenio-jobs/blob/master/setup.cfg#L40), which explicitly mentions `invenio-db[postgresql,mysql]` as test-dependency, despite `invenio-db` already being in its dependency-tree (as indirect dependency via `invenio-records-resources`, common dependencies are db-agnostic there too)